monitoring/prometheus/prometheus.yml

```
@@ -0,0 +1,104 @@
+  external_labels:
+    cluster: 'ecommerce-app'
+    environment: 'development'
+
+rule_files:
+  - "/etc/prometheus/rules/*.yml"
+
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets:
+          - alertmanager:9093
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)